cmd/selection_MCMC/selection_MCMC.go

```diff
@@ -0,0 +1,50 @@
+package main
```

Comment:
would be good to remove underscores.  To follow conventions I think you should consider "selectionMcmc/selectionMcmc.go"